setup/git-setup.md

```diff
@@ -38,4 +38,58 @@ passphrase again.
 
 #### Configuring git
 
-*To be supplied at a later date*
+Based on our workflow, we suggest that git is configured in specific
+ways to facilitate the way that we use the software.
+
+The mandatory settings (below) **must** be set for git to even
+function properly.
+
+The strongly suggested settings should generally be taken verbatim
+unless you have specific strong reasons for deviating, and you know
+what you're doing.
+
+The preference settings are useful to many, although your milage may
+vary.
+
+Finally, in common workflows with git, the aliases may make your work
+with git faster and easier.
+
+* Mandatory settings (**replace name/email with your own**):
+```
+git config --global user.name "Your Name"
+git config --global user.email youremailaddress@yourdomain.com
+```
+
+* Strongly suggested settings:
+```
+git config --global am.threeWay true
+git config --global apply.ignoreWhitespace change
+git config --global core.excludesfile ~/.gitignore
+git config --global log.decorate short
+git config --global pull.ff only
+git config --global push.default current
```

Comment:
The idea here was that you should do all of the "Strongly suggested settings". As I said at the beginning of this, *The strongly suggested settings should generally be taken verbatim unless you have specific strong reasons for deviating, and you know what you're doing.*

I think that's true for all of these, frankly. But some more than others.

Should I reword above, or just move these to mandatory because git will likely not do what you expect?